lib/games/addGamepass.js

```diff
@@ -0,0 +1,64 @@
+const http = require('../util/http.js').func
+const getGeneralToken = require('../util/getGeneralToken.js').func
+
+exports.required = ['universeId', 'name', 'priceInRobux']
+exports.optional = ['description', 'jar']
+
+// Docs
+/**
+ * 🔐 Create a gamepass.
+ * @category Game
+ * @alias addGamepass
+ * @param {number} universeId - The id of the universe.
+ * @param {string} name - The name of the gamepass.
+ * @param {number} priceInRobux - The price of the product.
+ * @param {string=} description - The description of the gamepass.
+ * @returns {Promise<GamepassAddResult>}
+ * @example const noblox = require("noblox.js")
+ * // Login using your cookie
+ * noblox.addGamepass(1, "A Gamepass", 100, "A cool item.")
+**/
+
+const nextFunction = (jar, token, universeId, name, priceInRobux, description) => {
+  const FormData = require('form-data');
```

Comment:
Don't require modules in functions like this.
This does not need a new dependency - I believe the new fetch API has a built in way of building these, and they are built manually elsewhere in the library.